cmd/qat_plugin/README.md

```diff
@@ -105,6 +105,12 @@ repository. Thus the easiest way to deploy the plugin in your cluster is to run
 $ kubectl apply -k 'https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/qat_plugin?ref=<RELEASE_VERSION>'
 ```
 
+NOTE: In case AppArmor kernel module is installed and enabled by default (Ubuntu, SUSE), use the customization overlay with AppArmor annnotations (otherwise plugin's daemonset will fail with bind/unbind errors):
+
+```bash
+$ kubectl apply -k 'https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/qat_plugin/overlays/apparmor_unconfined?ref=<RELEASE_VERSION>'
```

Comment:
this will not work anymore if the user is also interested in `Automatic Provisioning`. We will have to re-think the apparmor case thoroughly in #1575 before adding random notes.

Reply:
> this will not work anymore if the user is also interested in Automatic Provisioning

I re-tested, the annotation has no effect on provisioning, the provisioning is fine with or without it.

What connection this has to the automatic provisioning and why won't it work anymore?

> We will have to re-think the apparmor case thoroughly in #1575 before adding random notes.

We already have this overlay, also the annotation is used in e2e tests.

In what way is this comment random and what's preventing us to document an existing behaviour?

Reply:
> What connection this has to the automatic provisioning and why won't it work anymore?

that if they deploy `overlays/qat_initcontainer`, they won't get the annotation.

> In what way is this comment random and what's preventing us to document an existing behaviour?

it does not cover the helm/operator flows for example. I submitted #1575 so that the best approach to address the problem is agreed first